frontend/simple/js/events.js

```diff
@@ -25,24 +41,27 @@ export class GroupContract extends Events.HashableGroup {
         let index = state.invitees.findIndex(username => username === data.username)
         if (index > -1) {
           state.invitees.splice(index, 1)
-          state._async.push({type: 'HashableGroupAcceptInvitation', data})
+          Vue.set(state.profiles, data.username, {globalProfile: data.identityContractId, groupProfile: {}})
+          state._async.push('HashableGroupAcceptInvitation')
         }
       },
       // TODO: remove group profile when leave group is implemented
       HashableGroupSetGroupProfile (state, {data}) {
-        state.profiles[data.username] = _.merge(state.profiles[data.username], JSON.parse(data.json))
+        Vue.set(state.profiles[data.username], 'groupProfile', _.merge(state.profiles[data.username].groupProfile, JSON.parse(data.json)))
```

Comment:
The `.groupProfile` might not be set, and for that matter we need to verify that `state.profiles[data.username]` is even set here too.